### PR TITLE
Change markers for conservation projects

### PR DESCRIPTION
**Pull request by prachigarg19**
Change conservation markers style to: 
image

### CODE REVIEW

File: src/features/projects/components/maps/Markers.tsx [hunk 1]
```diff
@@ -45,16 +45,13 @@ export default function Markers({
...
-        ? 'conservationMarker'
-        : 'topUnapproved';
     return project.isTopProject && project.isApproved
       ? 'topApproved'
       : project.allowDonations
-      ? topUnapprovedType
+      ? 'topUnapproved'
       : 'notDonatable';
   };
+  console.log(searchedProject.properties);
```
Comment (norbertschuler) [inline src/features/projects/components/maps/Markers.tsx:54]: You should remove this again if it works as expected.

File: src/features/projects/styles/ProjectsMap.module.scss [hunk 2]
```diff
@@ -738,10 +738,6 @@
   background-color: #828282;
 }
 
-.conservationMarker {
```
Comment (norbertschuler) [inline src/features/projects/styles/ProjectsMap.module.scss:741]: This style is still used at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/src/features/projects/components/maps/Location.tsx#L31

**Review by norbertschuler — CHANGES_REQUESTED** [review]
Please check the code where the deleted style is still used.

File: src/features/projects/styles/ProjectsMap.module.scss (hunk 2 above)
Comment (prachigarg19) [reply src/features/projects/styles/ProjectsMap.module.scss:741]: @mariahosfeld do I need to change markers used here as well?

**Review by norbertschuler — APPROVED** [review]
LGFM

**Pull request merged**